### PR TITLE
[master < T0539-DX] Various doc tasks

### PR DESCRIPTION
**Pull request by vpavicic**
- [x] How to guide on checking logs and modifying configuration
- [x]  Linking changelog with new documentation
- [x] Adding missing DEFAULTS to the Configuration page
- [x] Fixing insolation values
- [x] Adding memory control to reference guide
- [x] Explaining average degrees
- [x] fixing Streams -> Transformation overview
- [x] Fix -> graph stream processing tutorial
- [x] mage/usage/loading-modules -> adding options to a table
- [x] added examples for BFS to Under the hood -> Graph algorithms
- [x] rewrote Social network analysis with NetworkX

### CODE REVIEW

File: docs/reference-guide/streams/transformation-modules/overview.md [hunk 1]
```diff
@@ -89,42 +89,6 @@ The file is now inside your Docker container.
 
 </details>
 
-:::caution
```
Comment (g-despot) [inline docs/reference-guide/streams/transformation-modules/overview.md:92]: Why did you remove this? @vpavicic

Comment (vpavicic) [reply docs/reference-guide/streams/transformation-modules/overview.md:92]: It was accidentally duplicated 
image

**Review by g-despot — APPROVED** [review]

**Review by g-despot — APPROVED** [review]
